xun/tests/test_stores.py

```diff
@@ -127,6 +131,7 @@ def test_store_driver_is_collection(cls):
         assert driver['c'] == 2
 
         assert set(driver.keys()) >= {'a', 'b', 'c'}
+        if  hasattr(store.driver, 'cache'): store.driver.cache.close()
```

Comment:
Goes for all of them

Reply:
The same as before.
I'm not sure how to tackle the problem differently. Are there any suggestions?

Reply:
DiskCache removed